src/repositories/SaleRepository.js

```diff
@@ -112,7 +112,8 @@ export const SaleRepository = {
   },
   SaleZoneLevel: {
     qry: service + 'qryZOL',
-    set2: service + 'set2ZOL'
+    set2: service + 'set2ZOL',
+    get: service + 'getZOL'
```

Comment:
remove get, not used

Reply:
it is not added by me it came with the merge from master